server/src/assetComputeDevTool.js

```diff
@@ -137,6 +137,11 @@ class AssetComputeDevTool {
         );
     }
 
+    async checkEventJournal(journalUrl) {
+        console.log("calling asset-compute-client#checkEventJournal");
```

Comment:
Maybe remove the log statement if it is only for debugging, when you're done with the story.